package.json

```diff
@@ -35,8 +35,11 @@
     "server": "python network-api/manage.py runserver",
     "start": "docker-compose up",
     "test:procfile": "node test/test-procfile.js",
+    "test:eslint:js:pre": "prettier \"cypress/integration/*.js\" \"source/js/**/*.js\" \"source/js/**/*.jsx\" webpack.config.js --write",
     "test:eslint:js": "eslint --config ./.eslintrc.json \"cypress/integration/*.js\" \"source/js/**/*.js\" \"source/js/**/*.jsx\" webpack.config.js",
+    "test:eslint:a11y:pre": "prettier \"source/js/**/*.jsx\" --write",
     "test:eslint:a11y": "eslint --config ./.eslintrc.a11y.json \"source/js/**/*.jsx\"",
+    "test:scss:pre": "prettier \"source/sass/**/*.scss\" \"source/js/**/*.scss\" \"network-api/networkapi/{,!(frontend)/**/}*.css\" --write",
```

Comment:
do we need to care about `.css` files?

Reply:
I thought it was `pre` as in "run before something else"` but then I saw the prettier command and dont know which is which now. Could we make `:pre` something more explicit, possibly?

Reply:
@mmmavis we have a few bare .css files in overrides, we might as well format them

@KalobTaulien no, you got that right: this runs "pre" the actual `test:scss` running, blindingly applying code formatting before running code _linting_

Reply:
Perfect. Thanks for adding that clarification Pomax!

Reply:
Ah I forgot we have a few `.css`. Good call including them.